docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/_index.md

```diff
@@ -76,12 +76,41 @@ Blog: [Using YugabyteDB xCluster DR for PostgreSQL Disaster Recovery in Azure](h
 
 For more information on the YugabyteDB xCluster implementation and its limitations, refer to [xCluster implementation limitations](../../../architecture/docdb-replication/async-replication/#limitations).
 
+## Schema Change mode for xcluster DR
+
+YBA supports setting up xcluster DR in either of two schema change modes, which differ in the steps required to replicate DDL operations from the DR primary to the DR replica.
+
+### Manual Schema Changes
+
+In this mode, table and index-level schema changes must be performed in a specific order on 
+1. The DR primary universe
+1. The DR replica universe, and
+1. In some cases, they must also be updated on the YugabyteDB Anywhere DR configuration.
+
+The exact sequence of these operations for each type of Schema Change (DDL) is described at [Manage tables and indexes](./disaster-recovery-tables/).
+
+### Semi-automated Schema Changes (Early Access)
+
+In this mode, table and index-level schema changes operations must be performed in the same order on
```

Comment:
Do we want to be more specific and say that each DDL change must be applied to both universe one by one?

Reply:
I see that we are more specific on the `Manage tables and indexes` page.

Reply:
Yes, here we just want to explain at a high level.